### ISSUE TITLE
How to pass through different limit for suggestions?

### ISSUE THREAD

**Issue by grtjn** [joemfb/ml-search-ng#90]
And potentially other params as well. Maybe through suggestOptions. If object and contains params, use that to extend var params?

**Comment by joemfb** [comment]
I guess we could use the same signature as `values()`: https://github.com/joemfb/ml-search-ng/blob/7d9fceac6464987d8568ebbc87ef69144cde0af8/src/ml-search.service.js#L1237

**Comment by grtjn** [comment]
Yes, makes sense. But there is already a second arg I think. Backwards compat might be a bit tricky..

By the way, I was also looking for a way to pass through extra request params to search, for instance to pass through view=facets..
